extensions/amp-date-countdown/1.0/base-element.js

```diff
@@ -1,10 +1,29 @@
 import {parseDateAttrs as parseDateAttrsBase} from '#core/dom/parse-date-attributes';
+import {dict} from '#core/types/object';
 
 import {PreactBaseElement} from '#preact/base-element';
 
+import {getTemplateElement, getTemplateFunction} from '#utils/template-utils';
+
 import {BentoDateCountdown} from './component';
 
-export class BaseElement extends PreactBaseElement {}
+export class BaseElement extends PreactBaseElement {
```

Comment:
We need to sanitize the result on the AMP layer. In the meantime, it's okay if you disable the feature on AMP to submit this PR.